pkgs/by-name/do/docfd/package.nix

```
@@ -14,26 +14,24 @@ ocamlPackages.buildDunePackage rec {
   };
 
   nativeBuildInputs = [ python3 dune_3 makeWrapper ];
-  buildInputs = [ poppler_utils ] ++
+  buildInputs = [ poppler_utils fzf ] ++
```

```suggestion
  buildInputs = [ ] ++
```

If these are not linked against, they don't have to be in `buildInputs`, adding them to the wrapper is enough.

---

> If these are not linked against, they don't have to be in `buildInputs`, adding them to the wrapper is enough.

Thanks, I should have realized that! Much appreciated.